routing/cross_mwm_connector_serialization.hpp

```diff
@@ -70,15 +73,16 @@ class CrossMwmConnectorSerializer final
       m_frontPoint = serial::LoadPoint(src, codingParams);
 
       BitReader<Source> reader(src);
-      m_osmId = reader.ReadAtMost64Bits(bitsPerOsmId);
+      // @todo
+      m_crossMwmId = reader.ReadAtMost64Bits(bitsPerOsmId);
       m_featureId = ReadDelta<decltype(m_featureId)>(reader) - 1;
       m_segmentIdx = ReadDelta<decltype(m_segmentIdx)>(reader) - 1;
       m_roadMask = base::asserted_cast<VehicleMask>(reader.ReadAtMost32Bits(bitsPerMask));
       m_oneWayMask = base::asserted_cast<VehicleMask>(reader.ReadAtMost32Bits(bitsPerMask));
       m_forwardIsEnter = reader.Read(1) == 0;
     }
 
-    uint64_t GetOsmId() const { return m_osmId; }
+    CrossMwmId GetCrossMwmId() const { return m_crossMwmId; }
```

Comment:
is it possible to return CrossMwmId by const ref?

Reply:
Yes. It's possible now for CrossMwmConnector::GetCrossMwmId(Segment const & segment) and for Transition::GetCrossMwmId(). Let's do that.